app/invocation/invocation_model.tsx

```diff
@@ -147,6 +147,10 @@ export default class InvocationModel {
       } else {
         model.failed.push(buildEvent as build_event_stream.BuildEvent);
       }
+
+      if (buildEvent.aborted && buildEvent.aborted.reason.toString().toLowerCase() == "skipped") {
```

Comment:
Do we still want to store this as an `aborted` event if it's skipped? That way you also shouldn't need the check in invocation.tsx.

Reply:
I put that check when storing aborted events and removed the check in invocation.tsx